README.adoc

```diff
@@ -277,7 +277,9 @@ Except the Frontend one, all these plugins publish a `reportsDirectory` property
     </plugin>
 ```
 |https://plugins.jenkins.io/junit/[Jenkins JUnit Plugin]
-|`withMaven(options: [junitPublisher(disabled: true)],...)`
+|`withMaven(options: [junitPublisher(disabled: true)],...)` +
+Since 3.11.0 more fine granular options: +
+`withMaven(options: [junitPublisher(fingerprintFilesDisabled: true, archiveFilesDisabled: true)],...)`
```

Comment:
you target the wrong publisher, changes must be on line 256

Reply:
Thank you for the hint. I have corrected it.